_includes/resources.html

```diff
@@ -2,12 +2,24 @@
   <div class="container">
     <div class="row">
       <div class="col-lg-12 text-center">
-        <h2 class="section-heading">Resources:</h2>
-        <h3 class="section-subheading text-muted">Ways to learn to code outside of UofT Coders!</h3>
+        <h2 class="section-heading">Friends and Resources</h2>
+        <h3 class="section-subheading text-muted">Groups and resources for coding and research</h3>
       </div>
     </div>
     <div class="col-lg-12">
       <ul>
+	    <li>
+          <a class='eventTitle' href="http://software-carpentry.org/lessons/">Software Carpentry</a>
+          <div class='eventSub'>Introductory lessons for various computing for research topics.</div>
+        </li>
+		<li>
+          <a class='eventTitle' href="http://internetresearchnetwork.org/">Internet Research Network</a>
+          <div class='eventSub'>An interdisciplinary network of researchers at UofT studying how the rise of the Internet is changing society, politics and the economy–and vice versa.</div>
```

Comment:
Yes, it should not be an en-dash. You should be able to write out the escaped version `&mdash;` or you can copy/paste —.